app/views/catalog/_constraints_element.html.erb

```diff
@@ -13,7 +13,8 @@
       <span class="filter-name"><%= label %></span>
     <% end %>
     <% unless value.blank? %>
-      <%= content_tag :span, value, class: 'filter-value', title: value %>
+      <% title = value.gsub(/<|>/, '') # the title could be html %>
```

Comment:
If this is for stripping html tags would it be better to use the standard `strip_tags`?  Or does `content_tag` already do that stripping and you're trying to partially avoid it?

Reply:
`strip_tags` is exactly what is needed here. Thanks.